Pollo/Views/Cards/MCResultCell.swift

```diff
@@ -14,10 +14,13 @@ class MCResultCell: UICollectionViewCell {
     // MARK: - View vars
     var checkImageView: UIImageView!
     var containerView: UIView!
+    var dotView: UIView!
     var highlightView: UIView!
     var innerShadow: CALayer!
     var numSelectedLabel: UILabel!
     var optionLabel: UILabel!
+    var selectedDotView: UIView!
+    var selectedImageView: UIImageView!
```

Comment:
this is outside the scope of this PR but can we make an issue to make variables `private let` // `private` where possible